package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "s47_weirdy_foody",
+  "version": "1.0.0",
+  "description": "Weirdly Foody I got this idea from social medial specially food vloggers they used to take out different content from the weird food items so I thaught to make a site where they can see weird food items name and their pics and some Youtube videos where youu can see the reaction after eating that food and honest reviews of different peoples. There will be a specific section in which ingredients of that food will be present.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
Consider adding the "dependencies" and "devDependencies" fields to your package.json file. This will help other developers understand what packages are necessary to run and develop your project. [important]